### PR TITLE
add new keys to the ocs-operator-config configmap

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: controllers/ocsinitialization/ocsinitialization_controller.go
Comment (iamniting) [reply controllers/ocsinitialization/ocsinitialization_controller.go:360]: We will always need these keys as we are exporting them to the rook operator deployment. CI is broken due to these missing keys.

**Review by Nikhil-Ladha — APPROVED** [review]
/lgtm

File: controllers/ocsinitialization/ocsinitialization_controller.go
Comment (iamniting) [reply controllers/ocsinitialization/ocsinitialization_controller.go:360]: Actually that is handled in the rook-ceph-operator-config configmap not in the `ocs-operator-config` configmap

Comment (jarrpa) [reply controllers/ocsinitialization/ocsinitialization_controller.go:360]: Don't resolve my conversations, man. If you're just going to shove them in there then at least inform the people working on this feature that you're doing this and explain why you're doing it.

**Comment by jarrpa** [comment]
Per conversation elsewhere, this is a temporary fix to unblock the CI due to a bug introduced with deploying rook-ceph-operator via operator bundle. The correct implementation of this is coming with PR #2475 (specifically [here](https://github.com/red-hat-storage/ocs-operator/pull/2475/files#diff-206c96d5cbb86076c508aea5cd093ce8306b159f5d969e5c8ea1496623b24efaR384-R394)).

/hold cancel

File: controllers/ocsinitialization/ocsinitialization_controller.go
Comment (iamniting) [reply controllers/ocsinitialization/ocsinitialization_controller.go:360]: my bad. I was trying to get this PR in ASAP to unblock CI issue.

**Comment by iamniting** [comment]
It is not required as #2475 was merged before this.

**Pull request closed without merging**
